aiohttp/client_reqrep.py

```diff
@@ -191,8 +191,8 @@ def __init__(self, method, url, *,
             url2 = url.with_query(params)
             q.extend(url2.query)
             url = url.with_query(q)
-        self.url = url.with_fragment(None)
         self.original_url = url
```

Comment:
Also, this code before my commit looks strange to me:

```python
        self.url = url.with_fragment(None)
        self.original_url = url
```

Both `url` and `original_url` are not modified after (at least by the object itself), so they are always the same. 

And then `original_url` is passed as `url` for ClientResponse. 

So, wasn't it actually intended to keep URL fragment in response object?